project/plugins.sbt

```diff
@@ -7,38 +7,18 @@ libraryDependencies ++= Seq(
   "org.joda" % "joda-convert" % "1.7",
 )
 
-resolvers ++= Resolver.sonatypeOssRepos("releases") ++ Seq(
-  Classpaths.typesafeReleases,
-  Resolver.typesafeRepo("releases"),
-)
+resolvers ++= Resolver.sonatypeOssRepos("releases")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.19")
+addSbtPlugin("org.playframework" % "sbt-plugin" % "3.0.0")
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.3.1")
+addSbtPlugin("com.github.sbt" % "sbt-native-packager" % "1.9.16")
```

Comment:
Actually not sure if sbt-native-packager needs to be pulled in explicitly because Play should provide it anyway.